files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -22,7 +22,7 @@
 # it below; otherwise that task will fail.
 #
 - file:
-    path: /export/home
+    path: /home
```

Comment:
since we're changing the user's home directory location here, we might want to use this file as the place that we change the mountpoint for the home directory dataset too..

I have some concerns if we actually change the mountpoint "live" on the running systems, due to potential processes currently using `/export/home`..

we might want to only update `/etc/fstab` to point to the new location, while leaving the existing `/export/home` mountpoint intact.. and then leverage a future reboot to "move" the mountpoint from `/export/home` to `/home`.. i.e. just mount at `/home` after the reboot..

but, if we do that, and change the user's home directory to be `/home`, we might run into conflicts.. e.g. `/export/home` will be mounted, while the user expects it to be in `/home`.. at least until the next reboot..

might need to think about how to do the "transition" properly, while processes are still running..

Reply:
Resolved this by adding the `preinst` script as per our discussions. The mount point for home dataset is being changed to `/home` during upgrade and that too we have done by checking the existence  in the `/etc/fstab`, if this entry exist with `/export/home` .